zazu/repo/commands.py

```diff
@@ -81,12 +89,93 @@ def clone(ctx, repository_url, nohooks, nosubmodules):
 
 
 @repo.command()
+@click.option('-i', 'interactive', is_flag=True, help='starts interactive repo init')
 @click.pass_context
-def init(ctx):
-    """Initialize repo directory structure"""
-    raise NotImplementedError
-
+def init(ctx, interactive):
+    """Initialize repo directory structure
+    TODo:
+    -if not git repo offer to make one
+    -if git repo, assume user wants current repo to use zazu
+    """
+    
+    def _get_plugin_list(plugin_subclass):
+        """helper function to pull lists of plugins"""
+        plugins = straight.plugin.load('zazu.plugins', subclasses=plugin_subclass)
+        known_types = {p.type().lower(): p.from_config for p in plugins}
```

Comment:
We may want to put fns like these in config.py...or make a plugins.py module better to have a common function that returns the dict then ask it for keys rather than a function that just returns the keys.